.github/ISSUE_TEMPLATE/1_bug_report.yml

```
@@ -53,6 +54,14 @@ body:
       placeholder: "e.g., 1.7.5.2 or 1.7.8.0"
     validations:
       required: true
+  - type: input
+    id: prestashop-installation-method
+    attributes:
+      label: How you installed PrestaShop
+      description: Choose the method you used to install PrestaShop.
+      placeholder: "e.g., downloading the package and where you downloaded it, using git clone, the docker image, etc."
+    validations:
+      required: true
```

```suggestion
      required: false
```

Maybe this should be optional?